### PR TITLE
[MRG] fixing the pseudo inverse test.

### PR DESCRIPTION
**Pull request by eolivi**
- correct failing test due to non-initialized memory
- use a sparseMatrix to store the diagonal S in the svd.
- return the transposed V in the svd

### CODE REVIEW

**Comment by agramfort** [comment]
can you put back the test so we can see if it fixes the pb?

File: OpenMEEG/libs/OpenMEEGMaths/tests/full.cpp [hunk 1]
```diff
@@ -108,33 +109,38 @@ int main () {
...
+        std::cout << "U :" << std::endl;
+        U.info();
+        std::cout << "S :" << std::endl;
+        S.info();
+        std::cout << "W :" << std::endl;
+        W.info();
         std::cerr << "Error: SVD is WRONG-1" << std::endl;
         exit(1);
     }
-    /* temporarly removing the pseudo inverse test, as casuing troubles on travis
```
Comment (eolivi) [inline OpenMEEG/libs/OpenMEEGMaths/tests/full.cpp]: I remove the "/*" and "*/"so the test is back on

File: OpenMEEG/libs/OpenMEEGMaths/tests/full.cpp [hunk 2]
```diff
@@ -108,33 +109,38 @@ int main () {
...
+    zero = M1*M1pinv*M1-M1;
     if (zero.frobenius_norm()>eps) {
+        std::cout << "pinverse = M^{+} = " << std::endl;
+        M1pinv.info();
+        std::cout << "M*M^{+}*M-M = zero matrix = " << std::endl;
         zero.info();
         std::cerr << "Error: PseudoInverse is WRONG-2" << std::endl;
         exit(1);
     }
-    */
```
Comment (eolivi) [inline OpenMEEG/libs/OpenMEEGMaths/tests/full.cpp]: end here

**Comment by agramfort** [comment]
awesome ! thx @eolivi

**Pull request merged**